backend/editor/api.py

```diff
@@ -387,8 +389,8 @@ async def import_from_github(request: Request, branch: str, taxonomy_name: str):
     if not await taxonomy.is_branch_unique():
         raise HTTPException(status_code=409, detail="branch_name: Branch name should be unique!")
 
-    result = await taxonomy.import_from_github(description)
-    return result
+    background_tasks.add_task(await taxonomy.import_from_github(description))
```

Comment:
```suggestion
    background_tasks.add_task(taxonomy.import_from_github, description)
```

Reply:
Thanks @eric-nguyen-cs. But with 
`background_tasks.add_task(taxonomy.import_from_github, description)`
it seems that the task is never really done, It seems to send only an empty list to the frontend whereas with 
`background_tasks.add_task(await taxonomy.import_from_github(description))`
I can see all the logs from taxonomy-editor-taxonomy_api while the taxonomy is parsed.
So to me it seems to be more complex.

Reply:
@Piv94165 
`background_tasks.add_task(await taxonomy.import_from_github(description))` calls the function but does not execute it in background (in Python we resolve arguments first)